test/images/echoserver/Dockerfile_windows

```diff
@@ -19,21 +19,24 @@ ARG REGISTRY
 FROM --platform=linux/amd64 alpine:3.6 as prep
 
 ADD https://openresty.org/download/openresty-1.13.6.2-win64.zip /openresty-win64.zip
+ADD http://wiki.overbyte.eu/arch/openssl-1.1.1i-win64.zip /openssl.zip
 RUN mkdir /openresty &&\
-    unzip /openresty-win64.zip -d /openresty
+    unzip /openresty-win64.zip -d /openresty &&\
+    mkdir /openssl &&\
+    unzip /openssl.zip -d /openssl
 
-FROM $REGISTRY/windows-image-builder-helper:1.1-windows-amd64-1809 as helper
 FROM $BASEIMAGE
 
 COPY --from=prep /openresty/openresty-1.13.6.2-win64 /openresty
-COPY --from=helper /Windows/System32/vcruntime140.dll /Windows/System32/
-COPY --from=helper ["/Program Files/OpenSSL", "/Program Files/OpenSSL"]
+COPY --from=prep /openssl /openssl
 
-ENV PATH="C:\openresty\;C:\bin\;C:\curl\;C:\Windows\system32;C:\Windows;C:\Program Files\PowerShell;"
+ADD ["https://raw.githubusercontent.com/openssl/openssl/OpenSSL_1_1_1-stable/apps/openssl.cnf", "/Program Files/Common Files/SSL/openssl.cnf"]
```

Comment:
is this available off of a tag? (is this likely to change in master?)

Reply:
That's taken from the ``https://github.com/openssl/openssl/tree/OpenSSL_1_1_1-stable`` branch, but we can probably use the ``https://github.com/openssl/openssl/tree/OpenSSL_1_1_1i`` tag instead.